### ISSUE TITLE
General low quality of text on learn/microcontrollers/5v-3v3

### ISSUE THREAD

**Issue by slavicd** [arduino/docs-content#589]
It's possible the translation on this page has gone wrong. The text is very hard to follow and there are many grammar issues. Is there an original that can be offered in order to improve the translation?

**Comment by marqdevx** [comment]
Hey!
The original is the one shown in English 😄 
You can make suggestions by opening a PR though

**Comment by marqdevx** [comment]
Also it is being updated a bit at #595
